init.c

```diff
@@ -87,6 +88,10 @@ static int early_mounts()
 	if (ret < 0)
 		exit_error(errno, "Could not mount /sys");
 
+	ret = mount("none", "/sys/kernel/security", "securityfs", 0, NULL);
```

Comment:
some systems might not have securityfs support built into kernel ... so have this a config that is default on?

Reply:
Ok, got it, I gonna add a config to this, something like `system.mount.sys.kernel.security` as a boolean and move this code to `other_mounts()` in order to get the config system ready

Reply:
New key added to config